include/evmc/evmc.hpp

```diff
@@ -160,9 +204,92 @@ class host
                   const uint8_t* data,
                   size_t data_size,
                   const evmc_bytes32 topics[],
-                  size_t topics_count) noexcept
+                  size_t topics_count) noexcept final
     {
         context->host->emit_log(context, &address, data, data_size, topics, topics_count);
     }
 };
+
+/// Abstract class to be used by Host implementations. Hides ::evmc_context vtable.
```

Comment:
Documentation here could be better. `evmc_context` doesn't have vtable.

Better explicitly point out, that this class should be derived when implementing Host